packages/cozy-stack-client/src/mangoIndex.js

```diff
@@ -3,7 +3,7 @@ import get from 'lodash/get'
 import isEqual from 'lodash/isEqual'
 
 /**
- * @typedef {object} MangoPartialFilter
+ * @typedef {Object} MangoPartialFilter
```

Comment:
Is it really `Object`?

Reply:
You can do what you want... But please be patient, something bigger is coming ... ;)

Reply:
O is certainly bigger than o, but I feel like you're talking about something else...